library/CloudBlobHelpers.cs

```diff
@@ -28,26 +28,24 @@ namespace Microsoft.Azure.Toolkit.Replication
     using System.Threading;
     using System.Threading.Tasks;
     using System.Security;
-    using Microsoft.WindowsAzure.Storage;
-    using Microsoft.WindowsAzure.Storage.Table;
-    using Microsoft.WindowsAzure.Storage.Blob;
     using System.Collections.ObjectModel;
+    using global::Azure.Storage.Blobs;
+    using global::Azure.Storage.Blobs.Models;
+    using global::Azure;
+    using global::Azure.Data.Tables;
 
     public class CloudBlobHelpers
     {
-        public static CloudBlockBlob GetBlockBlob(string configurationStorageConnectionString, string configurationLocation)
+        public static BlobClient GetBlockBlob(string configurationStorageConnectionString, string configurationLocation)
         {
-            CloudStorageAccount blobStorageAccount = CloudStorageAccount.Parse(configurationStorageConnectionString);
-            CloudBlobClient blobClient = blobStorageAccount.CreateCloudBlobClient();
-
-            CloudBlobContainer container = blobClient.GetContainerReference(GetContainerName(configurationLocation));
-            if (container.CreateIfNotExists())
+            var blobClient = new BlobServiceClient(configurationStorageConnectionString);
```

Comment:
Can you please put in the type here explicitly instead of var, helps with readability of the code